config/authors/tkko.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Tornike",
+  "bio": "",
```

Comment:
Software Engineer/Developer · Experienced in Flutter, Dart, Android, iOS, and Web, with a great sense of UX/UI design.